packages/wallets/src/hooks/useInputATMFormatter.ts

```diff
@@ -45,25 +45,31 @@ const useInputATMFormatter = (inputRef: React.RefObject<HTMLInputElement>, initi
         }
     }, [caret, formattedValue, caretNeedsRepositioning, input]);
 
-    const onChange = useCallback(
-        (e: DeepPartial<React.ChangeEvent<HTMLInputElement>> | React.ChangeEvent<HTMLInputElement>) => {
+    const checkExceedsMaxDigits = useCallback(
```

Comment:
future improvement (out of scope for this PR): would be really really nice to have some sort of structure/architecture around validation.
E.g. have a directory /utils/validatiors and have simple functions there:
- maxLengthExceeded
- tooBigValue 
etc etc

Reply:
so then its super easy to fix validation/update validation etc,